### PR TITLE
chore: create documentation improvement issue template

### PR DESCRIPTION
**Pull request by rouille**
This is an issue template for contributors who wish to improve the documentation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation-improvement.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (jon-hagg) [inline .github/ISSUE_TEMPLATE/documentation-improvement.md:1]: Needs an emoji

File: .github/ISSUE_TEMPLATE/documentation-improvement.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+labels: documentation
+assignees: danielolsen, rouille, BainanXia, jon-hagg, ahurli
+
+---
+
+### Problem
+Tell us about the confusion introduced in the documentation, what is lacking, etc.
+* I found [...] to be unclear because [...]
+* [...] made me think that [...] when really it should be [...]
+* There is no example showing how to do [...]
```
Comment (BainanXia) [inline .github/ISSUE_TEMPLATE/documentation-improvement.md:16]: Do you think we should further categorize the problems related to documentations, such as:
- Ambiguous
- Missing
- Outdated
- Wrong

Comment (rouille) [reply .github/ISSUE_TEMPLATE/documentation-improvement.md]: Done

**Review by jon-hagg — APPROVED** [review]
LGTM

**Review by BainanXia — APPROVED** [review]
:pray:

**Pull request merged**